### ISSUE TITLE
entry formalities to CERN for sprint 2

### ISSUE THREAD

**Issue by myrmoteras** [plazi/arcadia-project#210]
@jbenito3 @lnielsen @slint can you please arrange the entry formalities for the external participants at the Sprint 2, January 30- Feb 3?

https://github.com/plazi/arcadia-project/blob/master/sprint%202%2020230130.md#arcadia-team

**Comment by slint** [comment]
Yes, we'll make an Indico event, and share here the link for registration like the previous time.

**Comment by slint** [comment]
@myrmoteras the event for access has been set up at https://indico.cern.ch/event/1244607/registrations/91143/

Everyone participating in the sprint that needs access to the CERN-site has to register.

**Comment by flsimoes** [comment]
Mine is done

**Comment by JonasBlanco** [comment]
Done.

**Comment by tcatapano** [comment]
Done

**Comment by myrmoteras** [comment]
done

**Comment by myrmoteras** [comment]
@slint shouldn't we get an email with a link to print out the batch we need for the entry to CERN? So far I only got the confirmation that I registered, but not the link.
thanks for looking into this
Donat

**Comment by slint** [comment]
I've just granted access to the 9 people who've registered already, so you should receive emails shortly (let me know here if not).

From the 11 counted on the GitHub table that will attend on-site, I see Punnet (ping @punkish) and Peter missing.

**Comment by punkish** [comment]
I have not any email with link to register. Please resend the invitation to me. Thanks

**Comment by myrmoteras** [comment]
> I have not any email with link to register. Please resend the invitation to me. Thanks

@punkish please check the begin of this issue (3rd comment)

**Comment by peter-cornwell** [comment]
Eric Decker and i have both received access tickets
